sources/editor/Stride.Core.Assets.Editor/View/TemplateProviders/DictionaryNumberKeyTemplateProvider.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) .NET Foundation and Contributors (https://dotnetfoundation.org/ & https://stride3d.net) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
```

Comment:
No need for `and Silicon Studio Corp. (https://www.siliconstudio.co.jp)` in new files.

Reply:
OK, I have removed it!